book/src/themes.md

```diff
@@ -89,6 +89,14 @@ Less common modifiers might not be supported by your terminal emulator.
 | `hidden`       |
 | `crossed_out`  |
 
+### Inheritance
+
+Extend upon other themes by setting the `inherits` property to an existing theme.
+
+```toml
+inherits = "boo_berry"
```

Comment:
It would also be useful to see examples of overriding values from the inherited theme. For example

```toml
# Override the theming for "keyword"s:
"keyword" = { fg = "gold" }

# Override colors in the palette:
[palette]
berry = "#2A2A4D"
```